src/auth/auth-token-error-code.ts

```diff
@@ -2,6 +2,6 @@ export enum BBAuthTokenErrorCode {
   Unspecified = 0,
   NotLoggedIn,
   InvalidEnvironment,
-  PermissionScopeNoEnvironment,
+  PermissionScopeNoContext,
```

Comment:
We can't change the enum value's name here as it would constitute a breaking change.